tests/run-make/crate-name-help/issue-113981.b.rs

```diff
@@ -0,0 +1 @@
+pub fn main() {}
```

Comment:
Please check if this could also be a ui test. I'm certain I've accidentally written filenames like that before in ui tests and had to edit them, maybe it just all works out?

Reply:
I tried to use it as a ui test, we have a `tidy` check before `git push`:

```console
tidy error: Stray file with UI testing output: "/Users/yukang/rust/tests/ui/suggestions/issue-113981.b.stderr"
```

Reply:
Oh lol, that might be caused by rust-tidy assuming the `b` in `issue-113981.b.stderr` to be the *revision* of the test.

Reply:
Right and there's no suitable `// ignore-tidy-xxx` for this case. Apparently, you can place `ignore-tidy` in the file path to make tidy ignore it? Not sure how that is supposed to work. E.g. see `tests/ui/parser/shebang/issue-71471-ignore-tidy.rs`. You might be able to do something similar

Reply:
thanks for the tip, just made a quick fix 🤔
https://github.com/rust-lang/rust/pull/114058/commits/ab36d0e39f128778d98a3d528d93a355bc1d8c92